style.css

```diff
@@ -121,6 +118,8 @@ body {
   display: flex;
   flex-direction: column;
   align-items: center;
+  margin-left: 24px;
```

Comment:
[OPTIONAL] I know you have made an effort aligning the design. But it is still off. Kindly apply the recommended setting given below, that if this works on your side too?

| Your Design   | Edited Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221275367-5609b7ca-e581-4a54-84b7-4c53c860bd1c.png)| ![image](https://user-images.githubusercontent.com/61361037/221275246-32a85832-43ed-46e0-9773-a0530193d529.png)|